OCPBUGS-11113 making clarification about add the annotation cpu-quota… (#61043)
openshift/openshift-docs: modules/cnf-configuring-workload-hints.adoc

----
<1> If `highPowerConsumption` is `true`, the node is tuned for very low latency at the cost of increased power consumption.
<2> Disables some debugging and monitoring features that can affect system latency.

[NOTE]
====

> With this `PerformanceProfile` and `realTime` set to `true` the pod specification needs to include the annotation `cpu-quota.crio.io: "disable"`.

Reviewer: ```suggestion
If `realTime` is set to `true` in a performance profile, include the `cpu-quota.crio.io annotation in the pod specification and set it to `"disable"`.
```
I do not know if this changes the technical accuracy, but I thought this might be a bit tighter.

Author: I'll take that thanks, guess it was not clear to me if that is just how it is set, and it should not be set to `false`.

> When the `realTime` flag is set to `true` in a performance profile, include the `cpu-quota.crio.io: disable` annotation in the pod's YAML. Without this annotation the performance of the process within the pod is expected to be degraded.

Reviewer 2: I would actually tell the user to add the annotation to every guaranteed pod with pinned cpus. 

Realtime workload hint (call it out explicitly to not confuse the user wrt realtime kernel) makes it worse, because it adds the nohz_full kernel argument.
